src/test/javascript/spec/component/exam/manage/student-exams/student-exam-detail.component.spec.ts

```diff
@@ -255,4 +299,38 @@ describe('StudentExamDetailComponent', () => {
         // therefore no useful assertion about a concrete value is possible here
         expect(studentExamDetailComponent.studentExam.submissionDate).not.toBe(undefined);
     });
+
+    it('should update the percent difference when the absolute working time changes', () => {
+        studentExamDetailComponent.ngOnInit();
+
+        studentExamDetailComponent.workingTimeFormValues.hours = 4;
+        studentExamDetailComponent.updateWorkingTimePercent();
+        expect(studentExamDetailComponent.workingTimeFormValues.percent).toBe(100);
+
+        studentExamDetailComponent.workingTimeFormValues.hours = 3;
+        studentExamDetailComponent.updateWorkingTimePercent();
+        expect(studentExamDetailComponent.workingTimeFormValues.percent).toBe(50);
+
+        studentExamDetailComponent.workingTimeFormValues.hours = 0;
+        studentExamDetailComponent.updateWorkingTimePercent();
+        expect(studentExamDetailComponent.workingTimeFormValues.percent).toBe(-100);
+
+        // small change, not a full percent => should remain 0
+        studentExamDetailComponent.workingTimeFormValues.hours = 2;
```

Comment:
Should we really handle this case this way? I think it's important that if there is even the slightest deviation, it should be marked as such because with exams it can become spicy if someone gets even a bit more time than someone else.

Reply:
I now updated it to round it to two digits after the separator.